doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -705,3 +706,45 @@ For Network intensive instances, VNF Component should fit into a single NUMA zon
 ### 4.4.3. Transaction Volume Considerations
 
 Storage transaction volumes impose a requirement on North-South network traffic in and out of the storage backend. Data availability requires that the data be replicated on multiple storage nodes and each new write imposes East-West network traffic requirements.
+
+
+<a name="4.5"></a>
+## 4.5 Cloud Topology and Control Plane Scenarios
+
+Typically, Clouds have been implemented in large (central) data centers with 100' to tens of thousads of servers. Telco Operators have also been creating intermediate data centers in central office locations, colocation centers, and now edge centers at the physical edge of their networks because of the demand for low latency and high throughput for 5G, IoT and connected devices (inclding autonomous driverless vehicles and connected vehicles). Chapter 3 of this document, discusses [Cloud Topology](./chapter03.md#3.5) and lists 3 types of data centers: Large, Intermediate and Edge.
+
+For ease of convenience, unless specifically required, in this section we will use Central Cloud Center, Edge Cloud Center and Intermediate Cloud Center as representative terms for cloud services hosted at centralsied large data centers, Telco edge locations and for locations with capacity somehwre in between the large data centers and edge locations, respectively. The mapping of various terms, including the Reference Model terminology specified in Table [8-5](../../..//ref_model/chapters/chapter08.md#8.3.5) is as follows:
+
+- Central Cloud Center: Large, Regional Data Center
+- Intermediate Cloud Center: Metro Data Center
+- Edge Cloud Center: Edge, Mini-/Micro-Edge
```

Comment:
I wonder how this relates to the "edges" that are defined in the Edge Glossary which is part of the State of the Edge project under LF Edge: https://github.com/State-of-the-Edge/glossary/blob/master/edge-glossary.md ?

There's a diagram in the 5G section in the white paper that is also referenced below: https://www.openstack.org/use-cases/edge-computing/edge-computing-next-steps-in-architecture-design-and-testing/

Does the Intermediate Cloud Center cover the Aggregation Edge layer or is there any other parallel that we could draw?

Reply:
@ildikov Both the Intermediate and the Edge can behave as Aggregation edge -- it will depend upon the capacity and capabilities. The Access Edge would always be the Edge. Let em make changes. Thanks

Reply:
@pgoyal01 Sounds good, thanks for clarifying.